.env.sample

```
@@ -8,4 +8,10 @@ VITE_REACT_APP_MAP_TILER_API_KEY=
 # PostHog environment variables - only needed for prod
 VITE_PUBLIC_POSTHOG_KEY=
 VITE_PUBLIC_POSTHOG_HOST=
-VITE_ENVIRONMENT=local # local | dev | prod
\ No newline at end of file
+VITE_ENVIRONMENT=local # local | dev | prod
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_ENVIRONMENT key should go before the VITE_PUBLIC_POSTHOG_HOST key

(UnorderedKey)